scripts/_setup_hosts_file.sh

```diff
@@ -6,20 +6,21 @@ set -u
 set -e
 
 HOSTS_FILE=/etc/hosts
-touch ${HOSTS_FILE} #only useful if we are 
+touch ${HOSTS_FILE} #only useful if we are
```

Comment:
if we are what?

Reply:
unsure.

Reply:
Likely that was my half formed thought - feel free to remove that comment

Reply:
@lewisdaly I'll need your checkmark again since I removed it :)